requirements.txt

```diff
@@ -1,7 +1,7 @@
 numpy
 accelerate>=0.20.3
 packaging
-transformers>=4.34.1
+transformers>=4.34.1,<4.38.0
```

Comment:
#53 is merged so you shouldnt need this cap, thanks

Reply:
@Ssukriti Should we keep the cap for the transformers package avoid un intended errors like `xla_fsdp_v2`. We could create github workflow to run tests and then update the cap regularly.

Reply:
we dont need to keep static version , but yes in optional dependencies PR , @gkumbhat is looking into how to cap and we may cap to next major release. Now that CI/CD with automatically pull new release versions , if we see failing builds, we will update accordingly
the errors we were seeing with `xla_fsdp_v2` was actually due to code we wrote , which was good to catch and fix . It was not a API change from transformers, but we were setting env variables incorrectly

Reply:
In general if there is a specific version that doesn't work, or has a bug ,then we can also ask pip to ignore that particular version. 

https://github.com/foundation-model-stack/fms-hf-tuning/pull/54#discussion_r1516271877